### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by DerekRoy**
Small changes to flow and grammar to improve readability of the document.

### CODE REVIEW

**Comment by Piedone** [comment]
Hi Derek, thank you for your contribution! I just wonder, how did you arrive here?

File: Readme.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
 
 
 The module's source is available in two public source repositories, automatically mirrored in both directions with [Git-hg Mirror](https://githgmirror.com):
 
 - [https://bitbucket.org/Lombiq/orchard-javascript.net](https://bitbucket.org/Lombiq/orchard-javascript.net) (Mercurial repository)
 - [https://github.com/Lombiq/Orchard-JavaScript.Net](https://github.com/Lombiq/Orchard-JavaScript.Net) (Git repository)
 
-Bug reports, feature requests and comments are warmly welcome, **please do so via GitHub**.
-Feel free to send pull requests too, no matter which source repository you choose for this purpose.
+Bug reports, feature requests, and comments are welcome, **please do so via GitHub**.
```
Comment (Piedone) [inline Readme.md]: Why did you remove warmly? We mean it :).

**Comment by DerekRoy** [comment]
> Hi Derek, thank you for your contribution! I just wonder, how did you arrive here?

I am a student in a technical writing class, and I was tasked with proof reading and editing documents in the real world for an assignment. I looked up java script and found this project, and after reading this file felt I could contribute.

File: Readme.md (hunk 1 above)
Comment (DerekRoy) [reply Readme.md]: In terms of technical writing, warmly is a word that is not necessary. In a read me file it is important that the reader be able to read through and find information quickly. While warmly expresses a good emotional sentiment it adds to the complexity of the document without contributing any information. If you would like warmly included in my recommended changes, I can edit the changes and put it back in.

Comment (Piedone) [reply Readme.md]: Well I don't think "warmly" adds too much complexity :). Yes, please add it back.

Comment (DerekRoy) [reply Readme.md]: Done

**Comment by Piedone** [comment]
Thanks!
